src/components/TooltipTrigger/index.tsx

```diff
@@ -13,8 +13,8 @@ interface ITooltipTriggerProps {
 const TooltipTrigger: React.FC<ITooltipTriggerProps> = ({
   size = 20,
   iconColor = '#FFFFFF',
-  tooltipBackground = '#FFFFFF',
-  tooltipFontColor = '#242A2E',
+  tooltipBackground = '#333333',
+  tooltipFontColor = '#FFFFFF',
```

Comment:
Directly modifying will cause all colors using TooltipTrigger to change.
<img width="448" alt="image" src="https://github.com/user-attachments/assets/e0a195ae-c56e-49fe-9748-98b3b3cf7926">

Reply:
`<TooltipTrigger tooltipFontColor={tooltipFontColor} tooltipBackground={tooltipBackground} content={t('options_locale_toolTip')} />`

Perhaps it is possible to set the values of its related colors during specific use, like this.

Reply:
Yes, pass the tooltip color as props may not be quite a good practice I think, can we extract the styles into the css, so option page can override the css style to customize its color?